mailqueue/migrations/0001_initial.py

```diff
@@ -50,7 +50,12 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='attachment',
             name='email',
-            field=models.ForeignKey(blank=True, to='mailqueue.MailerMessage', null=True),
+            field=models.ForeignKey(
```

Comment:
Your editing the `0001_initial` meaning anyone already running `django-mail-queue` can run `migrate` and won't have there database updated to represent the new model.

Reply:
setting a on_delete to models.CASADE (default) inside models.py will not create a new migration. If I don't edit this migration it will fail in django 2.0.

See:
https://docs.djangoproject.com/en/dev/releases/1.9/#foreignkey-and-onetoonefield-on-delete-argument

<quote>
Update models and existing migrations to explicitly set the argument.
</quote>